### PR TITLE
Add option to set billing phone number mandatory

### LINKED ISSUE #254: Billing phone number not mandatory after 1.2.0 update
V1.2.0 removes the mandatory requirement from the billing phone number.

### PR DESCRIPTION
**Pull request by bor0**
Fixes #254.

### CODE REVIEW

File: includes/class-wc-gateway-ppec-checkout-handler.php [hunk 1]
```diff
@@ -114,11 +114,13 @@ function checkout_init( $checkout ) {
...
 	 */
 	public function filter_default_address_fields( $fields ) {
+		$require_phone_number = wc_gateway_ppec()->settings->require_phone_number;
+
 		if ( method_exists( WC()->cart, 'needs_shipping' ) && ! WC()->cart->needs_shipping() ) {
 			$not_required_fields = array( 'address_1', 'city', 'state', 'postcode', 'country' );
 			foreach ( $not_required_fields as $not_required_field ) {
 				if ( array_key_exists( $not_required_field, $fields ) ) {
-					$fields[ $not_required_field ]['required'] = false;
+					$fields[ $not_required_field ]['required'] = 'no' !== $require_phone_number;
```
Comment (gedex) [inline includes/class-wc-gateway-ppec-checkout-handler.php]: @bor0 this will makes other fields (`address_1`, `city`, etc) becomes required.

Comment (gedex) [inline includes/class-wc-gateway-ppec-checkout-handler.php]: Here's how to test it:
 .
* Start checkout from cart page.
* When placing order, you'll get:

   screen shot 2017-06-07 at 1 04 29 am

Whether **Require Phone Number** is enabled or not, you'll get the same notice.

Comment (bor0) [inline includes/class-wc-gateway-ppec-checkout-handler.php]: You're right! PR updated.

**Comment by bor0** [comment]
@gedex one more round please. Thanks!

**Pull request merged**